jenkinsfile.groovy

```diff
@@ -462,6 +491,40 @@ pipeline {
       } //parallel
     } //stage deploy
 
+    stage ("Load data to DB") {
+      when  { expression { params.action == 'apply' }}
+      agent { node { label "${jenkinsCtrlNodeLabel} && ${account}" } }
+      steps {
+        script {
+          if (!repoFunctionsFactory.checkoutGitRepo(
+              github.cvr_app.url,
+              params.branch,
+              github.cvr_app.name,
+              globalValuesFactory.SSH_DEPLOY_GIT_CREDS_ID
+          )) {
+            failure("Failed to clone repository ${github.cvr_app.url}; branch: ${params.branch}")
+          }
+
+          dir ("${github.cvr_app.name}/${databaseScriptDir}") {
+            if (awsFunctionsFactory.awsCli("aws dynamodb update-table --region ${globalValuesFactory.AWS_REGION} --table-name cvr-${params.environment}-recalls --provisioned-throughput ReadCapacityUnits=500,WriteCapacityUnits=500")
+            .status) {
+              failure("Failure while increasing recalls table's read and write capasity")
+            }
+            if (sh (
+              script: "npm install && AWS_REGION=${globalValuesFactory.AWS_REGION} ENVIRONMENT=${params.environment} npm run loadData",
```

Comment:
Does this script do a full clear+load of data or does it do nothing when data is present? Maybe we could add a flag to skip this stage that someone could select if he is applying on top of an environment that is healthy? It wouldn't eat up our limits of rescaling dynamodb capacity

Reply:
It doesn't clear the data, I'll add a flag for skipping that stage - good idea